bson/_cbsonmodule.c

```diff
@@ -2935,6 +3030,7 @@ static PyMethodDef _CBSONMethods[] = {
     {"_element_to_dict", _cbson_element_to_dict, METH_VARARGS,
      "Decode a single key, value pair."},
     {"_array_of_documents_to_buffer", _cbson_array_of_documents_to_buffer, METH_VARARGS, "Convert raw array of documents to a stream of BSON documents"},
+    {"_test_long_long_to_str", _test_long_long_to_str, METH_VARARGS, "Test conversion of extreme and common Py_ssize_t values to str."},
```

Comment:
Great idea!

Reply:
Full credit to @1fish2! Are the Evergreen tests failing due to an issue in the PR? Can't see the details.